integrations/utils/commonUtils.js

```diff
@@ -116,7 +116,24 @@ function flattenJson(data) {
   return result;
 }
 
+/**
+ * Check whether the passed eventname is mapped in the config
+ * and return the mapped event name.
+ * @param {*} event
+ * @param {*} eventsHashmap
+ * @returns mappedEventName
+ */
+function setEventMappingFromConfig(event, eventsHashmap) {
+  // if the event name is mapped in the config, use the mapped name
+  // else use the original event name
+  if (eventsHashmap[event]) {
+    return eventsHashmap[event];
+  }
+  return null;
+}
+
```

Comment:
Do we need this util function? We can do this small check in code directly.

Reply:
SInce eventMappingFromConfig is common for all ad destinations, and maybe upcoming destinations also, so kept its utility function in `commonUtils`